src/runtime/hexagon_host.cpp

```diff
@@ -918,6 +931,34 @@ WEAK void halide_hexagon_power_hvx_off_as_destructor(void *user_context, void *
     halide_hexagon_power_hvx_off(user_context);
 }
 
+WEAK int halide_hexagon_set_remote_thread_params(void *user_context, int priority, int stack_size) {
+    int result = init_hexagon_runtime(user_context);
+    if (result != 0) {
+        return result;
+    }
+
+    debug(user_context) << "halide_hexagon_set_remote_thread_params\n";
+    if (!set_remote_thread_params) {
+        // This runtime doesn't support changing the thread params.
+        return -1;
```

Comment:
In other similar "optional" cases where the runtime function is missing, we return 0.